impl/src/main/java/dev/paseto/jpaseto/impl/DefaultPasetoV2LocalBuilder.java

```diff
@@ -62,7 +62,7 @@ public String compact() {
         }
 
         // 3
-        byte[] nonce = cryptoProvider.blake2b(payload, randomBytes);
+        byte[] nonce = cryptoProvider.blake2b(randomBytes, payload);
```

Comment:
Interface and implementations have been changed to use this order (as that is the order used by similar methods)
```suggestion
        byte[] nonce = cryptoProvider.blake2b(payload, randomBytes);
```